client/js/screens/SignIn/SignInContainer.js

```diff
@@ -36,4 +38,10 @@ class LoginContainer extends Component {
   }
 }
 
-export default LoginContainer;
+export default SignInContainer;
+
+SignInContainer.defaultProps = {
+  login: null,
+  storeSessionToken: null,
+  token: null
```

Comment:
these are not props. these are function arguments.

You should have all of your prop validations with .isRequired and if they are not required only then have default props.